modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css

```
@@ -0,0 +1,26 @@
+.evp-process .number-process{
```

Could you rename the class to:
```suggestion
.evp-process-title .number-process{
```

---

The same for the ones below